src/style.css

```diff
@@ -1,3 +1,74 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap';
+
+html {
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+  text-transform: uppercase;
+  justify-content: center;
+  padding: 0;
+  margin: 0;
```

Comment:
Avoid using tag names for styling. Instead, use classes to style your elements. This helps maintain specificity and reusability of your styles. Consider replacing the `nav ul` selector with a class selector, for example `.nav__list`. - CHECKLIST: Don't use tag names for styling (except `html` and `body`).